src/test/rustdoc-ui/feature-gate-doc_cfg_hide.rs

```diff
@@ -1,3 +1,4 @@
+// gate-test-doc_auto_cfg
```

Comment:
Please add a separate test showing that doc(cfg) isn't added unless the feature is enabled (preferably a revision test to make the difference easy to see).

Reply:
I can but I added this here for the feature gate, not for testing `doc_auto_cfg`. ;)